store/rootmulti/store.go

```diff
@@ -567,19 +570,37 @@ func (rs *Store) getStoreByName(name string) types.Store {
 // TODO: add proof for `multistore -> substore`.
```

Comment:
Lets update the godoc for the `proofs` special case?

Reply:
Wdym by that exactly? Updating the method comment or anything else?

`godoc` seems the be deprecated: https://pkg.go.dev/golang.org/x/tools/cmd/godoc

Reply:
Going to merge, will update this in another PR if I didn't get it right

Reply:
Oh, it just means adding a comment about the function